cfaforecastrenewalww/inst/stan/renewal_ww_hosp_site_level_inf_dynamics.stan

```diff
@@ -365,6 +365,7 @@ generated quantities {
 
   // Deterministic calculation of state level R(t) from incident infections
   // and the generation interval
-  rt = (state_inf_per_capita ./ convolve_dot_product(state_inf_per_capita, gt_rev_pmf, uot + ot + ht))[uot+1: uot + ot + ht];
+  rt = (state_inf_per_capita ./ convolve_dot_product(state_inf_per_capita,
+    reverse(append_row(0,generation_interval)), uot + ot + ht))[uot+1: uot + ot + ht];
```

Comment:
Do you definitely want to do this with `convolve_dot_product` and not `update_infectiousness`?

Reply:
I think so, update infectiousness is needed because it is tricky to do the convolution without an incident case curve, requires the initialization piece (from the exponential growth initial burn in period). 

But because we have the full incident infection time series, the calculation of R(t) should be  straightforward. The convolution works just like the other convolutions where we have a full time series and we convolve with a delay distribution. The kicker is just that those other delay distributions do have a value at index 0 (even if it is 0), whereas the GI pmf is assumed to start at day 1 not day 0.